src/ews_xsd.erl

```diff
@@ -155,7 +250,11 @@ parse_element(Element) ->
     #element{name=Name, type=Type, default=Default,
              fixed=Fixed, nillable=Nillable,
              min_occurs=MinOccurs, max_occurs=MaxOccurs,
-             parts=Children}.
+             parts=Children};
+maybe_ref(Qname, _Element) ->
+    %%Name = wh:get_attribute(Element, name),
+    %%io:format("maybe_ref(~p) Name: ~p~n~p~n", [Qname, Name, Element]),
```

Comment:
Same as above, remove or put behind -ifdef. This would require a `begin`/`end`, or something will complain about unused variables I'm pretty sure.